source/MaterialXRuntime/RtFileIo.cpp

```diff
@@ -219,7 +219,7 @@ namespace
         if (nodeType != MULTIOUTPUT)
         {
             // For single output nodes we can match the output directly.
-            PvtOutput* out = prim->getOutput(PvtAttribute::DEFAULT_OUTPUT_NAME);
+            PvtOutput* out = prim->getOutput(EMPTY_TOKEN);
```

Comment:
Remove trying to look for a hard-coded output. Reused this interface. If preferred could add in a getFirstOutput() or getOutputs() and size can be scanned but this seems the simplest.